mlir/test/Dialect/OpenMP/invalid.mlir

```diff
@@ -36,9 +36,9 @@ func.func @invalid_nested_wrapper(%lb : index, %ub : index, %step : index) {
         omp.terminator
       }
```

Comment:
Nit: I suppose it doesn't matter because the parent op's verifier fails before checking this one, but I think it makes sense to add the attribute to `omp.simd` as well here to make sure the only verifier error in this function is the one we are checking.